advanced/Scripts/query.sh

```diff
@@ -58,12 +61,68 @@ scanList(){
     esac
 }
 
+# function to count new domain entries since last gravity run
+CountNewDomains() {
+  newdomainentries=$(
+    sqlite3 "${gravityDBfile}" << EOSQL
+      ATTACH '${gravityOLDfile}' AS old;
+      SELECT count(DISTINCT domain) FROM gravity WHERE domain NOT IN (SELECT domain FROM old.gravity);
+EOSQL
+      )
+  echo -e "  ${INFO} ${newdomainentries} new domain(s) found."
+  exit 0
+}
+
+# function to find blocked entries in the FTL database,
+# present (new) in the latest (active) gravity database,
+# and NOT present in the old gravity database.
+FindNewBlockedDomains() {
+  # Check if the OLD database exists.
+  if [[ ! -f "${gravityOLDfile}" ]]; then
+    echo -e "  ${CROSS}  Old database ${gravityOLDfile} doesn't exist."
+    exit 1
+  else
+    # Check if the current database is more recent.
+    if [[ $(sqlite3 "${gravityOLDfile}" "SELECT value FROM info \
+              WHERE property IS 'updated';") \
+        > \
+          $(sqlite3 "${gravityDBfile}" "SELECT value FROM info \
+              WHERE property IS 'updated';") ]]; then
+      echo -e "  ${CROSS}  Old database ${gravityOLDfile} is more recent."
+      exit 1
+    fi
+    #Run the query
+    mapfile -t resultarray < <( sqlite3 -separator ' ' "${ftlDBfile}" << EOSQL
+      ATTACH '${gravityDBfile}' AS new;
+      ATTACH '${gravityOLDfile}' AS old;
+      SELECT DISTINCT queries.domain, '(' || adlist.address || ')' FROM queries
+      JOIN gravity ON new.gravity.domain = queries.domain
+      JOIN adlist ON adlist.id = new.gravity.adlist_id
+      WHERE queries.domain NOT IN (SELECT domain FROM old.gravity)
+      AND queries.domain IN (SELECT domain FROM new.gravity)
+      AND queries.status IN (1,9)
+      AND timestamp > (SELECT value FROM new.info WHERE property = 'updated');
+EOSQL
+      )
+    if [ "${#resultarray[@]}" -eq 0 ]; then
+      echo -e "  ${INFO} No new domains have been blocked yet."
+    else
+      for (( j=0; j<"${#resultarray[@]}"; j++ )); do
+        echo -e "  ${INFO} Blocked: ${resultarray[j]})"
```

Comment:
```suggestion
        echo -e "  ${INFO} Blocked: ${resultarray[j]}"
```

This is the source of the extra `)`. You can even commit the change here on Github.

Reply:
Thank you, committed and signed (hopefully) the suggested change.

What would of happened if I edited the file, using github desktop, and committed? Would the PR automatically been updated?

Reply:
Yes. When you commit locally on your desktop and push the changes upstream to your github repo this PR will be updated automatically.